okhttp/src/test/java/okhttp3/OkHttpTest.kt

```diff
@@ -0,0 +1,11 @@
+package okhttp3
+
+import org.junit.Test
+
+class OkHttpTest {
+    @Test
+    fun testVersion() {
+        val semVerRegex = Regex("^[0-9]+\\.[0-9]+\\.[0-9]+(-.+)?$")
+        assert(semVerRegex.matches(VERSION))
```

Comment:
instead of java assertions, can you replace with assertj?

```
import org.assertj.core.api.Assertions.assertThat
...
assertThat(VERSION).matches("[0-9]+\\.[0-9]+\\.[0-9]+(-.+)?")
```

Reply:
Yes, changed ✔